packages/berry-builder/sources/build-bundle.js

```diff
@@ -81,21 +87,21 @@ clipanion
           reject(err);
         } else {
           const erroredStats = [];
-
-          for (const stats of allStats)
-            if (stats.compilation.errors.length > 0)
-              erroredStats.push(stats.toString(`errors-only`));
-
-          if (erroredStats.length === 0) {
-            stderr.write(String(`Build succeeded at ${new Date()}\n`));
-          } else {
-            stderr.write(String(`Build failed at ${new Date()}\n`));
-          }
-
-          for (const stats of erroredStats) {
-            stderr.write(stats.toString(`errors-only`));
-            stderr.write(`\n`);
-          }
+          console.log(allStats)
```

Comment:
Whoops. Just a side note @arcanis -- I'm getting an error when watching via `yarn build:cli -w`

Reply:
I haven't had the time to give much love to `-w,--watch` lately, I always build the bundle from scratch. Quite possible something needs to be fixed there 🤔